### PR TITLE
Portfolio Desktop version

### PR DESCRIPTION
**Pull request by ishimwepati**
In this Portfolio Desktop Version many features have been implemented, some of them are :

- Forms,
- Grid and Flex,
- Background with more images,
- Google Font imported, 
- Media query implemented,
- Display command well used.

### CODE REVIEW

File: index.html
Comment (Efiamotu-1) [inline index.html:1]: - [ ] The highlighted text in your about me shown below is not malking use of the appropriate color as can be seen in the figma image below. please i kindly suggest you make the necessary changes before resubmitting. :+1: :sparkles:  
- figma design 
fct
- your design
fc-t

Comment (Efiamotu-1) [inline index.html:1]: - [ ] The button for each works card doesn't align well on mobile screens as provided in the figma template below please kindly make the required changes so your portfolio website. It is important that your design matches correctly the provided figma template as this shows your attention to details as a frontend developer and this is a good skill to have :+1: :sparkles: 
- figma design
cb
- your design
c-b

File: README.md
Comment (Efiamotu-1) [inline README.md:1]: - [ ] The link you added to your github is currently broken please fix it by providing the appropriate links in your readme file as this makes your readme look professional. :+1: :sparkles:

**Review by Efiamotu-1 — CHANGES_REQUESTED** [review]
Hi @ishimwepati , 

 

Good job so far! 👏. Your Code is Awesome! 💪

There are some **few** issues 🐛 that you still need to work 🔧 on though to go to the next project ⏭️ but you are almost there! 🏃
## Don't Worry 🙏

It's ok to have bugs in your code 🐛. Please, see this code review as a **chance** for you to **learn** practical coding skills and learn **new tips** that can improve your code quality 💯 and help you **get a better ** 💼.

_I am going to guide you on how to fix them down below. 🤓_

## Highlights :dart: 
- ✔️ Descriptive README
- ✔️ Descriptive PR title and description
- The code is top-notch! 💯 and has a huge potential.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

> **Note:** Please, don't Merge This Pull Request yet. Only Merge when it is Approved ✅.


## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [@Efiamotu-1](https://github.com/Efiamotu-1/)
- This is My **Twitter**: [@EFYAMOTU](https://twitter.com/EFYAMOTU)
- This is My **linkedIn**.[Habeeb Efiamotu Musa](https://www.linkedin.com/in/musa-habeeb/)
- This is My **Slack**: [Habeeb Efiamotu Musa](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html
Comment (ishimwepati) [reply index.html:1]: Ohh thanks a lot for this wonderful comment, I'm done working on it  #42526E!

Comment (ishimwepati) [reply index.html:1]: Ohh great, I have seen it and it has been fixed, I appreciate it.

File: README.md
Comment (ishimwepati) [reply README.md:1]: Ohh good, I have checked the links and the one which links to my profile was not working, I have fixed it, 🙏

**Review by skyv26 — APPROVED** [review]
## Hi @ishimwepati, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

## STATUS: APPROVED 🟢 



## Highlights
- Good **PR** title ✅ 
- Descriptive *PR* description ✅ 
- Implement the features ✅ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

## ⚠️  WARNING ⚠️ 
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

## Have any doubt ❓ 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @skyv26 in your question so I can receive the notification. You can also follow me on the below platforms

<table>
<tr>
<td>

<a href="https://github.com/skyv26">
   @skyv26
</a>

</td>
<td>

<a href="https://www.linkedin.com/in/skyv2022/">
   linkedin-skyv2022
</a>

</td>
<td>

<a href="https://twitter.com/vrma_aakash">
   @vrma_aakash
</a>

</td>
<td>

<a href="https://www.instagram.com/skybrel/">
   @skybrel
</a>

</td>
<td>

<a href="https://www.frontendmentor.io/profile/skyv26">
   @skybrel
</a>

</td>
</tr>
</table>

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**